packages/ui/src/components/Topic/components/CreateTopicWizard.tsx

```diff
@@ -84,11 +84,13 @@ export const CreateTopicWizard: React.FC<CreateTopicWizardProps> = ({
     onCloseCreateTopic && onCloseCreateTopic();
   };
 
+  const trimmedTopicName = topicName.trim();
```

Comment:
```suggestion
  const isTopicNameValid = topicName.trim() !== "";
```
 I mean, either you DRY it all or you don't